blocks/teaser-list/teaser-list.css

```diff
@@ -53,11 +53,12 @@
 }
 
 .teaser-list .teaser-card .teaser-link {
+  display: inline-block;
   text-decoration: none;
   font-family: var(--sans-serif-font-medium);
   font-size: var(--font-size-16);
   letter-spacing: .16px;
-  line-height: 120%;
+  height: var(--spacer-element-06);
```

Comment:
Shouldn't really use a spacer element var for height. Just add the actual px value you want here.